val.go

```diff
@@ -246,3 +246,51 @@ func (v *DurationVal) Quantile(quantile float64) (rv time.Duration) {
 	v.mtx.Unlock()
 	return rv
 }
+
+// RawVal is a simple wrapper around a float64 value without any aggregation
+// (histogram, sum, etc). Constructed using NewRawVal, though its expected usage is like:
+//
+//	var mon = monkit.Package()
+//
+//	func MyFunc() {
+//	  ...
+//	  mon.RawVal("value").Observe(val)
+//	  ...
+//	}
+type RawVal struct {
+	mtx   sync.Mutex
+	value float64
+	count float64
+	key   SeriesKey
+}
+
+// NewRawVal creates a RawVal
+func NewRawVal(key SeriesKey) *RawVal {
+	return &RawVal{key: key}
+}
+
+// Observe sets the current value
+func (v *RawVal) Observe(val float64) {
+	v.mtx.Lock()
+	v.value = val
+	v.count++
+	v.mtx.Unlock()
+}
+
+// Stats implements the StatSource interface.
+func (v *RawVal) Stats(cb func(key SeriesKey, field string, val float64)) {
+	v.mtx.Lock()
+	value := v.value
+	v.mtx.Unlock()
+
+	cb(v.key, "recent", value)
+	cb(v.key, "count", value)
```

Comment:
this should come from v.count

Reply:
Yup. Fixed. But I also refactored it to accept custom aggregations.